src/main/java/uk/gov/hmcts/cmc/claimstore/services/UserService.java

```diff
@@ -25,4 +33,8 @@ public GeneratePinResponse generatePin(final String name, final String authorisa
             authorisation.replace("Bearer: ", "")
         );
     }
+
+    public String generateServiceAuthToken(){
```

Comment:
Why user service generates service auth tokens? It does not sounds like users service responsibility. Please find better place for it.

Reply:
moved to new service